### PR TITLE
Extract ecl eps properties

### PR DESCRIPTION
**Pull request by joakim-hove**
Minor simplification for 3D refactor. The purpose of this is to safer/simpler be able to change the underlying 3D properties

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2165 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2165 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2165 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2165 please

File: opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp [hunk 1]
```diff
@@ -171,7 +82,7 @@ struct EclEpsScalingPointsInfo
 
     // maximum saturations
     Scalar Swu; // oil
-    Scalar Sgu; // gas
+    Scalar Sgu; // gasg
     Scalar Sowu; // oil for the oil-water system
```
Comment (totto82) [inline opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp]: was this intentionally

**Comment by joakim-hove** [comment]
> Do we have tests that exposes imbibition? 

Not as far as I know.

> look very carefully at the imbibition logic and it seems unchanged.

good.

File: opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp (hunk 1 above)
Comment (joakim-hove) [reply opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp]: No - I'll fix it

**Pull request merged**